src/components/customSearch/CustomSearchRow.tsx

```diff
@@ -0,0 +1,141 @@
+import DeleteIcon from '@mui/icons-material/Delete'
+import { Grid, IconButton, Typography } from '@mui/material'
+import { SelectChangeEvent } from '@mui/material/Select'
+import { useState } from 'react'
+import { useAppConfig } from '../../hooks/useAppConfig'
+import { SearchParameter } from '../../types/SimpleSearch'
+import SearchDateInput from '../search/SearchDateInput'
+import SearchSelectField from '../search/SearchSelectField'
+import SearchTextInput from '../search/SearchTextInput'
+
+interface CustomSearchRowProps {
+  parameter: SearchParameter
+  index: number
+  onChange?: (e: React.ChangeEvent<any>) => void
+  remove: <T>(index: number) => T | undefined
+  enableDelete?: boolean
+  enableCondition?: boolean
+  fieldGroupIndex: number
+  setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void
+}
+
+const CustomSearchRow: React.FC<CustomSearchRowProps> = ({
```

Comment:
Could we re-use the search row we used for the simple search ?

Reply:
done